packages/playground/windows/ExperimentalFeatures.props

```diff
@@ -8,6 +8,11 @@
   </PropertyGroup>
 
   <PropertyGroup Label="Unpackaged playground-win32" Condition="'$(SolutionName)'=='playground-win32'">
+    <!--
+      WARNING - DO NOT MAKE THE VERSION OF MICROSOFT.UI.XAML BE ANYTHING BUT PRERELEASE.
+      Playground-Win32 is an unpackaged win32 app, and needs to use Microsoft.UI.Xaml from a prerelease,
+      to be able to carry WinUI in-app instead of using the Framework Package
+    -->
     <WinUI2xVersion>2.7.0-prerelease.210913003</WinUI2xVersion>
```

Comment:
Is this property needed anymore?

Maybe we can just use `WinUIPackageVersion`?

Reply:
We could, at the risk of `WinUIPackageName` being wrong. Plus I don't think we expect either `WinUIPackageName` or`WinUIPackageVersion` to be set by users, as I don't think they are setup in the targets to only set if blank.